remix/app/routes/auth.sign-out.test.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * @jest-environment node
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { createCookieSessionStorage } from '@remix-run/node';
+import { Authenticator } from 'remix-auth';
+
+import { action } from './auth.sign-out';
+import { sessionFactory } from '../tests/factories/session.factory';
+
+jest.mock('./auth.sign-out', () => ({
+  action: jest.fn(),
+}));
+
+describe('Auth sign-out', () => {
+  const sessionMock = { ...sessionFactory };
+
+  const sessionStorage = createCookieSessionStorage({
+    cookie: { secrets: sessionMock.secrets },
+  });
+
+  (action as jest.Mock).mockReturnValue({
+    headers: {
+      location: '/auth/sign-in',
+      'Set-Cookie': sessionMock.session,
+    },
+  });
+
+  describe('POST /auth/sign-out', () => {
+    it('returns to /auth/sign-in', async () => {
+      const request = new Request('http://localhost:3400/auth/sign-out', {
```

Comment:
I was initially surprised why we need to pass an absolute URL here but then I found https://github.com/remix-run/remix/discussions/993#discussioncomment-4125164 (this should probably go into the insight section of the pull request for others to understand this "strange" bit 💡)

To avoid repetition and reduce maintenance issues in the long run, I recommend creating a test helper in `tests/helpers/request.ts` so that we can use something like `makeRequest('/auth/sign-out', 'POST') in tests.

Reply:
Yes, for the first I also got the error from that, then I just passed the absolute URL there.

Solved on: [d094998](https://github.com/nimblehq/js-framework-benchmark/pull/105/commits/d0949985162ee27c4eb6523b2f893a17a4d8b2bb)